content/applications/finance/fiscal_localizations/ecuador.rst

```diff
@@ -554,13 +554,100 @@ withholdings percentages apply.
    withholding of 70% in a new line with the same tax support, the system will allow you as long as
    the total of the bases matches the total from the :guilabel:`Vendor Bill`.
 
+eCommerce
+---------
+
+This :ref:`ATS Report <ecuador/ats>` module enables the features and configurations to:
+
+- Choose the SRI Payment Method in each payment method configuration.
+- Customers can manually input their identification type and identification number in the checkout
+  process on the website.
+- Automatically generate a valid electronic invoice for Ecuador at the end of the checkout process.
+
+Configuration
+~~~~~~~~~~~~~
+
+The following configurations are needed to effectively generate electronic invoices after the
+checkout process on the website.
+
+Website settings
+****************
+
+To enable automatic invoicing, navigate to :menuselection:`Website --> Configuration --> Settings:
+Invoicing` section and activate the option :guilabel:`Automatic Invoice`. With this option active,
+an invoice will be generated after the checkout process.
+
+.. tip::
+   The email template for the invoice can be modified from the :guilabel:`Invoice Email Template`
+   field under the :guilabel:`Automatic Invoice` option.
+
+.. important::
+   The sales journal used for invoicing will be the first in the sequence of priority in the
+   :guilabel:`Journal` menu.
+
+Payment providers
+*****************
+
+Activate the payment providers that will be used to capture payments through the eCommerce website.
+To do so, navigate to :menuselection:`Website --> Configuration --> Settings: Shop - Payment`
+section and then click on the :guilabel:`View other providers` button under the :guilabel:`Activate
+Payments` heading. From here, each payment provider can be configured by selecting a provider
+record. Once a provider is configured, be sure to click the :guilabel:`Publish` button so it is
+available in the checkout process when paying.
+
+Payment methods
+***************
+
+Activate the payment methods that each payment provider can have. To do this, click on the option
+:guilabel:`Enable Payment Methods` button within the :guilabel:`Configuration` tab of each provider.
+This will allow each provider to have one or more payment methods.
+
+When configuring the payment method, it is **mandatory** to set the :guilabel:`SRI Payment Method`
+for each method. This field will appear after you create and save the payment method for the first
+time.
+
+.. note::
+   The :guilabel:`SRI Payment Method` field will not be visible when configuring the payment method
+   for the first time. This information is mandatory to correctly generate the electronic invoice
+   from the eCommerce sale. To access that field, click on a **payment method** to access its
+   configuration menu.
```

Comment:
```suggestion
   Adding the :guilabel:`SRI Payment Method` is necesary to generate correctly the electronic invoice
   from an eCommerce sale. Select a **payment method** to access its configuration menu and the field.
```
I'm not sure this should be in a note

Reply:
@xpl-odoo Yes, it should be noted.

Reply:
I meant that it could be part of the core text; I wasn't sure using the note markup was relevant :)